docs/the-new-architecture/use-app-template.md

```diff
@@ -3,5 +3,61 @@ id: use-app-template
 title: Using the New App Template
```

Comment:
I was wondering if there might be a better way to title this. "New App Template" is not necessarily meaningful to an RN developer (externally, this just looks like I'm using the template for 0.68 and changing a configuration flag.)

I was thinking it could be "Creating a New Architecture app", but I'm wondering if there's more information we're trying to convey by describing this as using "the new app template".

This may also relate to my question below about the relation of this page to "Getting Started".

Reply:
Yup, that all makes sense, thank you!

Reply:
> I was thinking it could be "Creating a New Architecture app", but I'm wondering if there's more information we're trying to convey by describing this as using "the new app template".

+1 here. We always referred to "Using the New App Template" internally to quickly refer to this concept. However "Creating a New Architecture app" is more widely understandable and makes less assumptions.

Reply:
Feel free to change the title of the page @lindboe

Reply:
Updated!